Src/FluentAssertions.DataSets/FluentAssertions.DataSets.csproj

```diff
@@ -59,7 +59,7 @@
       <PackagePath>contentFiles\any\any\</PackagePath>
     </None>
   </ItemGroup>
-  
```

Comment:
What are these for?

```csproj
<PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
  <WarningsAsErrors>;NU1605;NU5104</WarningsAsErrors>
</PropertyGroup>

<PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
  <WarningsAsErrors>;NU1605;NU5104</WarningsAsErrors>
</PropertyGroup>
```

Reply:
Probably copy-pasted from the core library ;-)